user_guide/src/examples/dataframe/cat_join.py

```diff
@@ -0,0 +1,15 @@
+import polars as pl
+from .datatypes import df
+
+
+colors = pl.DataFrame({"id": [1, 2, 3], "name": ["blue", "red", "green"]})
```

Comment:
Can you add a trailing comma in the dict and behind it? That will force black multiline

Reply:
Will do 🫡. Do you ever use shorter line breaks? The commas feel very hacky.